docs/mobile-apps/faq.md

```diff
@@ -56,6 +56,13 @@ Yes. See [Camera Image Injection](/mobile-apps/features/camera-image-injection).
 * **Android emulators** - not supported.
 * **iOS simulators** - not supported.
 
+#### **Is there a maximum time I can use a real device for either manual or automated testing, provided I actually do interact with the device?**
+
+For real devices on the public cloud, the limit is 6 hours; for private cloud devices there is no limit.
+
+#### **Can I upload files to or download files from a real device (iOS/Android)?
```

Comment:
Hey @lauramthomas -- This heading is missing the `**` at the end to close the "bold" tag, but also, you don't need to add bold to these headings. Our H4 CSS definition already makes these headings bold, so these will look identical with or without the `**`.